src/sonic-yang-models/tests/yang_model_tests/yangTest.json

```diff
@@ -812,6 +812,48 @@
         }
     },
 
+    "DEVICE_METADATA_CORRECT_BUFFER_MODEL_PATTERN": {
+        "sonic-device_metadata:sonic-device_metadata": {
```

Comment:
I will suggest removing non-mandatory fields to keep the file small. 
As per below. We can remove all field except, "buffer_model": "dynamic"

Reply:
Done